tests/aws/services/ec2/test_ec2.py

```diff
@@ -170,110 +190,147 @@ def test_reserved_instance_api(self, aws_client):
         )
         assert 200 == rs["ResponseMetadata"]["HTTPStatusCode"]
 
-    @markers.aws.unknown
-    def test_vcp_peering_difference_regions(self, aws_client_factory, region_name):
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            # AWS doesn't populate all info of the requester to the peer describe until connection available
+            "$..pending-acceptance..VpcPeeringConnections..AccepterVpcInfo.CidrBlock",
+            "$..pending-acceptance..VpcPeeringConnections..AccepterVpcInfo.PeeringOptions",
+            # LS leaves as `[]`
+            "$..VpcPeeringConnections..AccepterVpcInfo.CidrBlockSet",
+            "$..VpcPeeringConnections..RequesterVpcInfo.CidrBlockSet",
+            # LS adds, not on AWS
+            "$..VpcPeeringConnections..AccepterVpcInfo.Ipv6CidrBlockSet",
+            "$..VpcPeeringConnections..RequesterVpcInfo.Ipv6CidrBlockSet",
+            # LS doesn't add
+            "$..VpcPeeringConnections..ExpirationTime",
+        ]
+    )
+    @markers.aws.validated
+    def test_vcp_peering_difference_regions(
+        self, aws_client_factory, region_name, cleanups, snapshot
+    ):
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.key_value("vpc1-id"),
+                snapshot.transform.key_value("vpc2-id"),
+                snapshot.transform.key_value("peering-connection-id"),
+            ]
+        )
         region1 = region_name
-        region2 = region_name  # When cross-region peering is supported, change to SECONDARY_TEST_AWS_REGION_NAME
-
-        # Note: different regions currently not supported due to set_default_region_in_headers(..) in edge.py
+        region2 = SECONDARY_TEST_AWS_REGION_NAME
```

Comment:
nit: I think you can avoid importing this config variable by using the `secondary_region_name` fixture, from the [multi account document](https://github.com/localstack/localstack/tree/master/docs/testing/multi-account-region-testing):
> Importing and using localstack.constants.TEST_ values is not advised.

I've launched a multi account run just to be sure 😄 as we're meddling in region territory here.

https://app.circleci.com/pipelines/github/localstack/localstack/25538/workflows/4147ddde-7bf2-41d7-ad04-0b45a28e5eb2

Reply:
Thanks for the pointer. I will switch to the appropriate fixture.

Reply:
I have updated the code to use the fixture and updated the snapshot to remove the reliance on the hard-coded `ap-southeast-1` . I am not rerunning the pipeline with `randomize-aws-credentials` as it did pass the first time and being that this is a cross region and not a cross-account test, the credentials shouldn't have an effect.